modes/vterm/evil-collection-vterm.el

```diff
@@ -62,6 +62,58 @@ also uses `evil-mode'."
                        "vterm"
                      "emacs"))))
 
+(defun vterm-evil-insert ()
```

Comment:
Nit: Might be good to have some comments on what this does. I'm guessing this gets us into insert mode at the actual point the cursor is on in the terminal instead of at the end of the line.

Reply:
Yes! I'll add comments soon.

Reply:
I added some comments. Let me know if these aren't sufficient!